js/components/index-page.js

```diff
@@ -8,8 +8,8 @@ export default class IndexPage extends PureComponent {
     return (
       <div className='index-page'>
         <h1>3D Plate Tectonics models</h1>
-        <h2>Use <a href={`${window.location.pathname}?authoring=true`} target='_blank' rel='noopener noreferrer'>
-          authoring mode
+        <h2>Use <a href={`${window.location.pathname}?planetWizard=true`} target='_blank' rel='noopener noreferrer'>
```

Comment:
I think "Use *the* planet wizard" is a little cleaner

Reply:
Thanks, fixed.